django/db/backends/mysql/base.py

```diff
@@ -104,6 +104,7 @@ class DatabaseWrapper(BaseDatabaseWrapper):
     data_types = {
         'AutoField': 'integer AUTO_INCREMENT',
         'BigAutoField': 'bigint AUTO_INCREMENT',
+        'SmallAutoField': 'smallint AUTO_INCREMENT',
```

Comment:
I put it here to keep the auto fields together because this isn't entirely alphabetical - `BigIntegerField` is further down. But we can fix order in a separate commit of you like?

Reply:
Yes only `BigInteger` breaks alphabetical order (in all backends), I don't want to break it more.

Reply:
> Yes only `BigInteger` breaks alphabetical order

Actually many more break the order than that. I've moved `SmallAutoField` between `SlugField` and `SmallIntegerField`.